test/resources/examples/generate.ts

```diff
@@ -0,0 +1,334 @@
+/* eslint-disable indent */
+import assert from "assert";
+import BN from "bn.js";
+import {
+  getExecutedSellAmount,
+  solutionObjectiveValueComputation,
+} from "../math.js";
+import {flat, dedupe} from "../array-shims.js";
+import {
+  TestCase,
+  TestCaseInput,
+  ObjectiveValueComputation,
+  ComputedSolution,
+  ComputedOrder,
+} from "./";
+
+/**
+ * Generates a test case to be used for unit and e2e testing with the contract
+ * with computed solution values and objective values.
+ * @param {TestCaseInput} input The input to the test case
+ * @param {boolean} [debug=false] Print debug information in case of
```

Comment:
noticed that the `strict` parameter is missing from the doc comments. This is probably an issue from before, just wanted to point it out.

Reply:
Yeah, will fix for now, but I'm generally not sure how realistic it is to keep these docstrings in sync unless there is some linter rule making sure everything is referenced correctly.

Reply:
I suggest getting rid of the type information and default value, as that is provided by the TS language service anyway.